contrib/dragonbox-cmake/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+set(DRAGONBOX_INCLUDE_DIR ${ClickHouse_SOURCE_DIR}/contrib/dragonbox/include)
+
+add_library(dragonbox INTERFACE)
```

Comment:
But there is cpp file to compile.

Reply:
We use only `dragonbox/dragonbox_to_chars.h` in our codebase, I thought that it is enough.